Apps/Src/FaultState.c

```diff
@@ -12,13 +12,26 @@
 #include "CANbus.h"
 #include "BSP_UART.h"
 #include "config.h"
-#ifndef SIMULATION
-#include "stm32f4xx.h"
-#else
+#include "BSP_PLL.h"
+#ifdef SIMULATION
 #include "Simulator.h"
 extern uint8_t stateCount;
 #endif
 
+void delay_u_copy(uint16_t micro)
```

Comment:
if you're going to copy over ```delay_u()```, make it static and make it more descriptive in naming (i.e. ```delay_u_faultstate()```)

Reply:
also a caution about the ```uint16_t``` parameter: you won't be able to have a delay higher than ~65.5ms

Reply:
fixed